src/libs/SidebarUtils.ts

```diff
@@ -127,13 +126,16 @@ function getOrderedReportIDs(
         // However, this code needs to be very performant to handle thousands of reports, so in the interest of speed, we're just going to disable this lint rule and add
         // the reportDisplayName property to the report object directly.
         // eslint-disable-next-line no-param-reassign
```

Comment:
Can we remove this line?

Reply:
Bump on this question

Reply:
Sorry I missed that we cant remove that because in this line we are reassigning the function parameter, if we want to do that we will need to refactor this function, but also there is a comment which is explaining why we are it was done

Reply:
The next line is `if (report) {`, right? Seems like two lines down we have the same disable?

Reply:
yeah you are right this one is not needed it was a leftover after some debugging and didnt see that, thanks !